File: app/src/main/java/com/saizad/mvvmexample/components/auth/AuthActivityViewModel.java
package com.saizad.mvvmexample.components.auth;

import com.saizad.mvvmexample.di.auth.AuthEnvironment;

import javax.inject.Inject;

public class AuthActivityViewModel extends AuthViewModel {

    @Inject
    public AuthActivityViewModel(AuthEnvironment authEnvironment) {
        super(authEnvironment);
    }
}

File: app/src/main/java/com/vm/frameworkexample/components/auth/AuthActivityViewModel.kt
package com.vm.frameworkexample.components.auth

import com.vm.frameworkexample.di.auth.AuthEnvironment
import dagger.hilt.android.lifecycle.HiltViewModel
import javax.inject.Inject

@HiltViewModel
class AuthActivityViewModel @Inject constructor(
    authEnvironment: AuthEnvironment
) : AuthViewModel(authEnvironment)